### DISCUSSION TITLE
Can't find the button to import all skins.

### DISCUSSION THREAD

**Discussion by Yazzah** [ppy/osu#22512] [Q&A]
title is self explanatory, did they change anything recently? I haven't kept up with lazer for a while.

**Comment by ItsJustSomeDude** [comment]
Scroll to the top of settings and use the Re-run initial setup Yellow button.  I think it imports at step 2 or 3.

**Reply by bdach** [reply]
Related: https://github.com/ppy/osu/discussions/21908

I've also seen a mention somewhere that the first setup overlay may be getting an easier way to navigate between steps quicker. Can't find where this was stated though.
